activetable.css

```diff
@@ -51,8 +56,37 @@ tr.odd td.right-answer {
 }
 input[type="text"] {
     font-size: 1em;
+    line-height: 1.5em;
     border: 0;
     padding: 0 10px;
     margin: 0;
-    background-color: rgba(0, 0, 0, 0);
+    box-sizing: border-box;
+    display: block;
+    width: 100%;
+    background-color: transparent;
+}
+input[type="text"]:hover {
+    outline: 2px solid #a0a0ff;
```

Comment:
@smarnach bolder maybe :) Basically, it needs good contrast to yellow (#ffffe0) as well (if not more than), and, more importantly, be easily distinguished from border color (#c0c0c0). It's a bit to much to ask from sinlgle color, so I would suggest making it bolder :)